scripts/link-validator.conf

```diff
@@ -64,6 +64,7 @@ site-link-validator {
     "http://localhost:8080"
     # broken URL in license report
     "http://pholser.github.com/"
+    "http://specs2.org/"
```

Comment:
Ah...I can't visit this website. Why add this? 
maybe
https://etorreborre.github.io/specs2/

Reply:
the license-report.html is generated - we don't really get to control what goes in it

that specs.org is what it is linking to and it now seems to be broken

so that means I need to get the validator to ignore it - that is what this setting is for

Reply:
Thank you for your explanation. It may be an issue with specs2, and there may be an issue with their domain DNS.

LGTM